src/test/java/net/starschema/clouddb/jdbc/JdbcUrlTest.java

```diff
@@ -197,6 +197,49 @@ public void canConnectWithApplicationDefaultCredentials() throws SQLException, I
     stmt.executeQuery("SELECT * FROM orders limit 1");
   }
 
+  @Test
+  public void canImpersonateServiceAccountWithApplicationDefaultAsSource()
+      throws IOException, SQLException {
+    Properties testProps = getProperties("/protectedaccount-json.properties");
+    String url =
+        BQDriver.getURLPrefix()
+            + testProps.getProperty("projectid")
+            + "/"
+            + testProps.getProperty("dataset");
+    String targetServiceAccount = "looker-service-account@looker-test-db.iam.gserviceaccount.com";
```

Comment:
This should be in one of the properties files rather than embedded directly in the code since it's part of Looker's test environment for this package.  If somebody else wants to test this library, they'll want to use a different service account for their test environment.